### ISSUE TITLE
По реєстрації:

### ISSUE THREAD

**Issue by P1ller** [essayNEXT/red_repo#4]

**Comment by P1ller** [comment]
Можна робити реєстрацію або просто через номер телефону (в телерам-апі є метод запиту номера) або через мейл і зберігати дані в БД

**Comment by Jaberwocy** [comment]
Але ж користувач вже залогінений - він заходить в бот зі свого аккаунту в Телеграм. Чи не так? Його дані мабуть слід вносити до БД - телефон або нік.

**Comment by P1ller** [comment]
не кожеш юзер відкриває свій номер телефону - може і заборонити доступ. А через бота можна його отримати, але тільки для бота =)

**Comment by Jaberwocy** [comment]
я теж про це. Але нащо номер телефону?
Для реєстрації нік або можна зробити прив'язку до аккаунта Гугл або Фейсбук.

**Comment by IgYa** [comment]
Вносити треба id юзера

**Comment by Jaberwocy** [comment]
На пошту можна робити розсилку та нагадування.

**Comment by Jaberwocy** [comment]
Так ІД треба вносити до БД. Але ми поки про реєстрацію, її краще все ж по ніку.

**Comment by P1ller** [comment]
Нікнейм можна змінити, тоді юзера не буде. А боту потрібно для надсилання повідомлення - ID

**Comment by Jaberwocy** [comment]
Так, з цим я згоден повністю - для БД та обліку юзерів ІД необхідний. Просто кажу про те, що при реєстрації зручніше вводити нікнейм або пошту. ІД не відразу й знайдешь. Хоча й телефон те ж можна.

**Comment by P1ller** [comment]
ID не потрібно знаходити, воно приходить боту автоматично з повідомленням юзера. І, відповідно, його можна записувати в БД
